src/openrct2/network/Network.cpp

```diff
@@ -1531,7 +1583,7 @@ void Network::Server_Send_MAP(NetworkConnection* connection)
         }
         return;
     }
-    size_t chunksize = 65000;
+    size_t chunksize = CHUNK_SIZE;
```

Comment:
This is not exactly the same value - is that not a problem?

Reply:
I think the reason this value was picked to be 65000 is because currently the size is encoded as uint16_t so it left some spare room for data before/after the chunk, it shouldn't be an issue at least nothing problematic showed up on my end. Now its exactly 63 kilobytes so enough spare room for anything else and still big enough

Reply:
Also added a comment to clarify that this shouldn't be changed to anything bigger.